cmd/tailscaled/tailscaled.go

```diff
@@ -60,6 +64,7 @@ func main() {
 
 	opts := ipnserver.Options{
 		StatePath:          *statepath,
+		AutostartStateKey:  "root",
```

Comment:
make "root" a named constant (unexported is fine) so you can clarify in attached docs that this is or is not related to the "root" Unix account. It might be worth a less ambiguous name if it's unrelated to user/group names. (I think it means "default" or "base" or "daemon" key?)

Reply:
Used `_daemon` because that's very un-username-ish, so keeps open the option of per-user state down the road.